packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -137,6 +137,12 @@ export function createEditPageNode({
             label: documentTypes[type].displayName,
         }));
 
+        React.useEffect(() => {
+            apollo.cache.evict({ fieldName: "pageTreeNodeSlugAvailable" });
+            // should only be executed on initial render
+            // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Is there a problem when passing apollo to the dependencies array?

Reply:
No, I added it